### ISSUE TITLE
Add support for WMF based player in core. 

### ISSUE THREAD

**Issue by ofTheo** [openframeworks/openFrameworks#5156]
Codec support is making our Directshow based player much, much harder to maintain ( see: https://github.com/openframeworks/openFrameworks/issues/5114#issuecomment-223786484 ) 

It would be good to look at adding the Windows Media Foundation player to the core which has native mp4 support:
https://github.com/DomAmato/ofxWMFVideoPlayer

Maybe keeping legacy support for the Directshow player as an option.

**Comment by danzeeeman** [comment]
WMF doesn't work on some Windows machines if they don't have a GPU that
supports hardware decoding. I hit this when trying to remotely deploy a 360
Video player app on Windows 7.

**Comment by ofTheo** [comment]
@danzeeeman ahh - good to know.  I wonder if there is a software fallback that can be enabled? Or we could always switch back to DS when WMF playback fails.

**Comment by danzeeeman** [comment]
Yeah but I couldn't get any MP4 container with an h264 codec to play with
DS I could only get a mpeg2 which looked terrible

**Comment by ofTheo** [comment]
@danzeeeman - that should be fixed with this codec pack ( https://github.com/openframeworks/openFrameworks/issues/5114#issuecomment-227208136 )

**Comment by danzeeeman** [comment]
dope!  I'll check it out!
